brainstem.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rr"
   gem.add_development_dependency "rspec", "~> 3.5"
   gem.add_development_dependency "sqlite3"
-  gem.add_development_dependency "mysql2"
+  gem.add_development_dependency "mysql2", "0.4.10"
```

Comment:
Why is the a strict requirement? I think we should be supporting up to `5.7.17-13` (since that's my workstation's version ;) )

Reply:
Some of our gems are not compatible with the latest `mySql2` gem version so had to lock it

Reply:
Oh, the gem! I connected the wrong dots. 

This is fine. Currently we're on `0.4.10` as well.